Cabal/src/Distribution/Simple/Setup.hs

```diff
@@ -201,6 +201,11 @@ data ConfigFlags = ConfigFlags {
     configPrograms_     :: Option' (Last' ProgramDb), -- ^All programs that
                                                       -- @cabal@ may run
 
+    configAppend        :: Flag Bool, -- ^specifies whether or not the @configure@
+                                      -- command should overwrite or append flags
+    configOverwrite     :: Flag Bool, -- ^specifies whether or not the @configure@
+                                      -- command should backup the configuration
+
```

Comment:
This is `Cabal`, but those are `cabal-install` flags, right? In that case, they should go in `cabal-install` (like eg. `v2-install` has its own additional `ClientInstallFlags`)

Reply:
What does that actually entails? Because it seems that a "second" `ClientInstallFlags` would just bloat everything, since it is everywhere within `cabal-install`. Is there any other option?

Reply:
> would just bloat everything, since it is everywhere within cabal-install

I'm not sure I understand what you mean. Wouldn't it be somewhat confined to the configure code, in the command-specific flags?

Reply:
I see what you mean. It's far more widespread than I thought.

Ideally we'd find a way to isolate the flags... but that's out of scope.

I guess you can put it in ConfigExFlags.

Reply:
Done.

Reply:
Thanks!

Reply:
Is this request stuck? It shows as resolved, but at the bottom it still points to it.

Reply:
I have no clue if "resolve conversation" counts, or if @fgaz needs to resolve or if this doesn't matter nad @fgaz needs to mark his interview as "approved".

Reply:
It was marked resolved by @fgaz, I unresolved and resolved again just to double check.

Reply:
I think I have to mark it as approved or dismiss the review. Let me take a quick look (the last time I stopped at the flags) and I'll do it